ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/versions/deneb/SignedBlockContentsSchema.java

```diff
@@ -28,7 +28,7 @@
       final SignedBlobSidecarsSchema signedBlobSidecarsSchema) {
     super(
         "SignedBlockContents",
-        namedSchema("signed_block_contents", signedBeaconBlockSchema),
+        namedSchema("signed_block", signedBeaconBlockSchema),
```

Comment:
we dont want contents here? I thought thats what we were adding in a bunch of places? does the name need to change too?

Reply:
Nope, this element is the `signed_block`, `SignedBlockContents` is the enclosing one